default.json

```diff
@@ -30,7 +30,8 @@
 				"matchUpdateTypes": [
 					"minor",
 					"patch"
-				]
+				],
+				"excludePackageNames": ["jsonrpsee"]
```

Comment:
Why are we doing this here and not in https://github.com/digicatapult/dscp-node/blob/main/renovate.json? Doing it here means any rust project we use will ignore that dependency